meta-refkit-core/recipes-ostree/refkit-ostree/files/refkit-ostree/src/refkit-ostree.c

```diff
@@ -0,0 +1,1642 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
```

Comment:
The latest version of the code has not been run through yet. I'll try to get that done. Our standard go-to-guy for static scans is on vacation. I will need to rebootstrap my you-know-which-scanner setup from scratch in my development machine and I suspect that will take a considerable amount of time. It'd be nice to get this thing moving forward already before and address any potential issues from the static analysis in a separate commit and PR.